test/support/movie_server.ex

```diff
@@ -16,7 +16,13 @@ defmodule PropCheck.Test.MovieServer do
   def start_link(), do:
     GenServer.start_link(__MODULE__, [], name: __MODULE__)
 
-  def stop(), do: GenServer.call(__MODULE__, :stop)
+  def stop() do
```

Comment:
Note that this function could also call `GenServer.stop/1`, which is synchronous. This is what's done in [movie_server.erl](http://proper.softlab.ntua.gr/code/movies/movie_server.erl) provides `movie_server:stop/0`.

Reply:
Good Point. I will change both implementations, it is much more to the point.